### PR TITLE
feat: move discord token to env var

### PR DESCRIPTION
**Pull request by samdammers**
Seen a few too many forks committing their config.json with their secret discord token to github.
Small fix to make that a bit easier, by default users won't do it accidentally unless they edit .gitignore

### CODE REVIEW

**Review by TannerGabriel — COMMENTED** [review]
Thank you for the great PR. I just left a minor comment I would like to address before merging.

File: README.md [hunk 1]
```diff
@@ -36,6 +36,9 @@ cd discord-bot/
 
 # Install the dependencies
 npm install
+
+# Configure Discord Bot Token
+echo "DISCORD_TOKEN='INSERT_YOUR_TOKEN_HERE'" > .env
```
Comment (samdammers) [inline README.md:41]:
```suggestion
 echo "DISCORD_TOKEN='INSERT_YOUR_TOKEN_HERE'" > .env
```

TIL

[the remark this answers is not in the document]
Comment (samdammers) [reply README.md]: TIL, have done (I think correctly) :ok_hand:

**Review by TannerGabriel — APPROVED** [review]
Thanks again!

**Pull request merged**